auth/resources/authserver.properties.sample

```diff
@@ -26,6 +26,8 @@ httpsPort=28051
 defaultHost=0.0.0.0
 logFilePath=/var/log/seagate/auth/server
 samlMetadataFileName=saml-metadata.xml
+maxAccountLimit=1001
```

Comment:
not required here since internal accounts need not to be mentioned here
with latest discussion we will keep this limit as per user specification and we will consider "s3InternalAccounts" list for calculating internal account  count.